tests/View/Blade/BladeCustomTest.php

```diff
@@ -52,20 +55,17 @@ public function testCustomShortStatements()
         $this->assertEquals($expected, $this->compiler->compileString($string));
     }
 
-    public function testValidCustomNames()
+    /**
+     * @dataProvider customNamesDataProvider
+     * @param string $name
+     * @throws \ReflectionException
+     */
```

Comment:
Format this DocBlock as detailed in the contribution guide: https://laravel.com/docs/8.x/contributions#phpdoc